src/tms_service.ts

```diff
@@ -110,6 +131,108 @@ export class TMSService extends AbstractEmsService {
     this._config = config;
   }
 
+  /*
+  This static function transforms a style to use the passed language
+  */
+  public static transformLanguage(
+    style: EmsVectorStyle,
+    lang: keyof typeof TMSService.SupportedLanguages
+  ): EmsVectorStyle {
+    const omtLang = TMSService.SupportedLanguages[lang]?.omtCode;
+
+    if (!omtLang) {
```

Comment:
nit: in general, I try to avoid using `(!var)` to check if something is undefined because it can also be triggered by `0` or `false`. 

```suggestion
    if (omtLang === undefined) {
```

Reply:
Totally agree, thanks! that code was removed so nothing to do (I reviewed the rest of the file changes as well).